django_project/social_users/models.py

```diff
@@ -0,0 +1,32 @@
+# -*- coding: utf-8 -*-
+import logging
+
+LOG = logging.getLogger(__name__)
+
+import itertools
+from django.contrib.auth.models import User
+from django.contrib.gis.db import models
+
+
+class UserLink(models.Model):
+    """
+    List of user link
+    """
+    link = models.CharField(blank=True, default="", null=True,
```

Comment:
please use 'pull left' for all python code we write i.e.:

```
 link = models.CharField(blank=True, default="", null=True, max_length=200)
```


```
 link = models.CharField(
    blank=True, default="", null=True, max_length=200)
```

See https://github.com/inasafe/inasafe/wiki/Coding-Standards#code-formatting for a complete set of the coding standards we should adhere to.